ACCESS-OM2-GMD-Paper-Figs/Fig12-ZonalTempSaltBias.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
```

Comment:
<p>How does one access an 8 worker, 192GB ARE instance? For reference, this next cell crashed my 28-core XXLarge instance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>mmm. Interesting. It worked for me. I think the 8 workers, 192GB is an old thing.</p>